Upgrade notification command does not copy properly due to box borders

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [X] I have read the console error message carefully (if applicable).

### Description


## Summary

It appears that the text does not copy properly from this docusaurus warning message. The file which generates this message is `packages/docusaurus/bin/beforeCli.mjs`. This *could* be a non-docusaurus specific problem, but it does not appear to be a shell problem (I tried with `zsh` and `bash`).

<img width="892" alt="Screenshot 2023-08-17 at 4 24 24 PM" src="https://github.com/facebook/docusaurus/assets/42948753/78ad63d3-b867-4845-a9c2-d068a2867acf">

I recently got this upgrade note, and wanted to upgrade my docusaurus version. I copy and pasted the command into my terminal, but the following happened:

```zsh
(base) sanderschulhoff@Sanders-MacBook-Pro Learn_Prompting % npm i @docusaurus/core@latest @docusaurus/plugin-client-redirects@latest @docusaurus/plugin-google-analytics@latest     │
   │          @docusaurus/plugin-ideal-image@latest @docusaurus/preset-classic@latest @docusaurus/theme-mermaid@latest           │
   │                                          @docusaurus/module-type-aliases@latest
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "│" of package "│": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in: /Users/sanderschulhoff/.npm/_logs/2023-08-17T23_20_04_692Z-debug-0.log
zsh: command not found: │
zsh: command not found: │
```

The `|` were quite weird as they didn't appear in the image I uploaded. I pasted the text elsewhere, and found that it pasted as:

```text
npm i @docusaurus/core@latest @docusaurus/plugin-client-redirects@latest @docusaurus/plugin-google-analytics@latest     │
   │          @docusaurus/plugin-ideal-image@latest @docusaurus/preset-classic@latest @docusaurus/theme-mermaid@latest           │
   │                                          @docusaurus/module-type-aliases@latest
```

The terminal (a zsh shell) could not properly handle it. I asked GPT-4 to reformat it, and it gave me the following, which ran properly.

```text
npm i @docusaurus/core@latest \
      @docusaurus/plugin-client-redirects@latest \
      @docusaurus/plugin-google-analytics@latest \
      @docusaurus/plugin-ideal-image@latest \
      @docusaurus/preset-classic@latest \
      @docusaurus/theme-mermaid@latest \
      @docusaurus/module-type-aliases@latest
```


### Reproducible demo

_No response_

### Steps to reproduce

1. Run these individually:
```
npm install @docusaurus/core@2.4.0 @docusaurus/preset-classic@2.4.0
npx docusaurus --version
```

2. Copy the command output (between the backticks):
<img width="892" alt="Screenshot 2023-08-17 at 4 39 05 PM" src="https://github.com/facebook/docusaurus/assets/42948753/bb1b8b1f-d749-495d-886d-a5cdd45b35b2">

3. Paste the command into the terminal and attempt to run it.

### Your environment

<img width="205" alt="Screenshot 2023-08-17 at 4 41 53 PM" src="https://github.com/facebook/docusaurus/assets/42948753/f765df45-0139-445c-927b-6709b7b7f063">

npx version: 9.7.2
node version: v20.4.0

VSCode, a bash or zsh terminal

### Self-service

--- a/packages/docusaurus/bin/beforeCli.mjs
+++ b/packages/docusaurus/bin/beforeCli.mjs
@@ -149,7 +149,7 @@ export default async function beforeCli() {
         top: '-',
         bottom: '-',
         left: ' ',
-        right: ' '
+        right: ' ',
       },
     };
 

--- a/packages/docusaurus/bin/beforeCli.mjs
+++ b/packages/docusaurus/bin/beforeCli.mjs
@@ -141,7 +141,16 @@ export default async function beforeCli() {
       margin: 1,
       align: 'center',
       borderColor: 'yellow',
-      borderStyle: 'round',
+      borderStyle: {
+        topLeft: ' ',
+        topRight: ' ',
+        bottomLeft: ' ',
+        bottomRight: ' ',
+        top: '-',
+        bottom: '-',
+        left: ' ',
+        right: ' '
+      },
     };
 
     const docusaurusUpdateMessage = boxen(

